### ISSUE TITLE
CME data from quandl is removed

### ISSUE THREAD

**Issue by bbqaaq** [t1user/zipline#1]
Thanks you so much for making this repo as a reference to me.

However as Im trying to run this repo, I found that CME data from quandl has been removed already.

May I ask if you will do some update for this?

**Comment by t1user** [comment]
Yes, unfortunately this data is no longer available on quandl. I haven't
updated the script to work with with any other data source mainly because I
haven't found one (good quality, free, etc).
